app/src/validators/object.js

```diff
@@ -37,7 +37,7 @@ const schema = {
   searchObjects: {
     query: Joi.object({
       objId: scheme.guid,
-      originalName: type.alphanum,
+      name: type.alphanum,
```

Comment:
This will fail on "foobar.pdf". We'll need to allow that. For now we'll reduce the specificity to just `Joi.string()`.